code/datums/mutations/passive.dm

```diff
@@ -11,3 +11,26 @@
 /datum/mutation/human/biotechcompat/on_losing(mob/living/carbon/human/owner)
 	owner.adjust_skillchip_complexity_modifier(-1)
 	return ..()
+
+/datum/mutation/human/good_looking
+	name = "Good Looking"
+	desc = "A long lineage of strong genes has made this person look somewhat good compared to most, although it does make them a bit vain most likely."
+	quality = POSITIVE
+	text_gain_indication = "<span class='notice'>You feel like you're looking good!</span>"
+	text_lose_indication = "<span class='notice'>You no longer feel like like a prince charming.</span>"
+
+/datum/mutation/human/good_looking/on_acquiring(mob/living/carbon/human/owner)
+	if(..())
+		return
+	owner.AddElement(/datum/element/beauty, 2500) //what a sigh to behold!
```

Comment:
most likely optional but is there any good way to make `2500` a define? i guess it's out of scope but there really is no reference for what values of beauty there are